src/Application/ApplicationFileProcessor.php

```diff
@@ -94,9 +100,16 @@ public function run(Configuration $configuration, InputInterface $input): Proces
         }
 
         if ($configuration->isParallel()) {
+            $this->fileDependenciesCache->cacheAllFiles($allFiles);
             $processResult = $this->runParallel($filePaths, $input, $postFileCallback);
         } else {
-            $processResult = $this->processFiles($filePaths, $configuration, $preFileCallback, $postFileCallback);
+            $processResult = $this->processFiles(
```

Comment:
We pass the list of all possible files as this is needed by the dependency resolver